demo/admin/src/dashboard/LatestBuilds.tsx

```diff
@@ -1,19 +1,21 @@
-import { gql } from "@apollo/client";
-import { useTableQuery } from "@comet/admin";
+import { gql, useQuery } from "@apollo/client";
 import { LatestBuildsDashboardWidget } from "@comet/cms-admin";
 import * as React from "react";
 
 import { GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables } from "./LatestBuilds.generated";
 
 export const LatestBuilds: React.FC = () => {
-    const tableQuery = useTableQuery<GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables>()(LATEST_BUILDS, {
-        resolveTableData: (data) => ({
-            data: data.builds,
-            totalCount: 5,
-        }),
-    });
+    const { data, loading, error } = useQuery<GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables>(LATEST_BUILDS);
 
-    return <LatestBuildsDashboardWidget tableQuery={tableQuery} />;
+    return (
+        <LatestBuildsDashboardWidget
+            data={{
+                rows: data?.builds,
+                loading,
+                error,
```

Comment:
Shouldn't loading and error be outside of data? This would be more in line with how these three variables are used by Apollo. Also, the error prop has been removed in MUI X v6: https://mui.com/x/migration/migration-data-grid-v5/#removed-props

Reply:
> Why can't you use the autoHeight prop? https://mui.com/x/react-data-grid/layout/#auto-height

Ah, I gave up after trying to set `height: auto` in the styling, didn't know `autoHeight` existed 😅 

> Shouldn't loading and error be outside of data? 

👍🏼 

> Also, the error prop has been removed in MUI X v6

As far as I know we still use v5 everywhere, so I'd wait with refactoring that until we update to v6.